env.sh

```diff
@@ -112,4 +117,4 @@ else
    echo "Sourcing $(realpath $SHELLFILE) for environment" 
    source $SHELLFILE
    echo "$(realpath ${SHELLFILE})" > $DIR/bin/current_shell_path
-fi
+fi
```

Comment:
```suggestion
fi

```
Just to minimize the number of unnecessary changes, so that the git history and the blame function gives more clear output.

Reply:
you can commit this suggestion by clicking on the `Commit suggestion` button above